logging/log.go

```diff
@@ -1,18 +1,50 @@
 package logging
 
 import (
+	"errors"
 	"fmt"
+	"io"
 	"log"
+	"os"
+	"time"
 
 	"github.com/logrusorgru/aurora/v3"
 )
 
+var logDir = "./logs"
 var logLevel = 0
```

Comment:
Just a formatting thing

```go
var (
    logDir = "./logs"
    logLevel = 0
)
```

Reply:
Addressed in commit [4bbb966](https://github.com/WiiLink24/wfc-server/pull/48/commits/4bbb96663419071f4abc63ffaa3526081035c403).